testing/scenario_app/lib/src/send_text_focus_semantics.dart

```diff
@@ -15,8 +15,14 @@ class SendTextFocusSemantics extends Scenario {
   /// Constructor for `SendTextFocusScemantics`.
   SendTextFocusSemantics(PlatformDispatcher dispatcher) : super(dispatcher);
 
+  bool _firstFrameDrawn = false;
```

Comment:
From https://github.com/flutter/flutter/issues/55778#issuecomment-1042516601
> It looks like the first frame of the test is supposed to fake a text field in `onBeginFrame` https://github.com/flutter/engine/blob/2962099077b36704071802ea5595f4a016a1c214/testing/scenario_app/lib/src/send_text_focus_semantics.dart#L39-L41
> 
> Then the test taps somewhere: https://github.com/flutter/engine/blob/2962099077b36704071802ea5595f4a016a1c214/testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m#L66-L67 Which triggers `onPointerDataPacket`, which "focuses" the fake text field: https://github.com/flutter/engine/blob/2962099077b36704071802ea5595f4a016a1c214/testing/scenario_app/lib/src/send_text_focus_semantics.dart#L108-L109
> 
> When I attach, I right after the tap I can see node 0 correctly being updated with `flutter::SemanticsFlags::kIsFocused`. https://github.com/flutter/engine/blob/2962099077b36704071802ea5595f4a016a1c214/lib/ui/semantics/semantics_update_builder.cc#L135
> 
> But then `onBeginFrame` is immediately called again (maybe because of the keyboard animating in?) and it resets node 0 back to an unfocused text field.
> 

The test passes when I make this change to only call the `onBeginFrame` logic to create the fake text input only once, the first time it's called.

Reply:
> The test passes 

Well, on my machine... it failed in CI.
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8822017057875020497/+/u/test:_Scenario_App_Integration_Tests/stdout